cli/main.cpp

```diff
@@ -20,7 +20,7 @@
 /**
  *
  * @mainpage Cppcheck
- * @version 2.10.99
+ * @version 2.11.99
```

Comment:
imho it should say 2.10.99. 2.11 has not been released yet.

Reply:
The dev versions always had the `.99` suffix - it basically means `2.11.dev`.

Reply:
hmm yes. I suggest that we change this. imho: `2.11.0 < 2.11.99`

Reply:
Yes - that has irked me for a while.

Off the top of my head I have no idea how to address this properly without changing the version scheme.

I propose we keep it consistent with the previous dev releases for now and address this until the next release. I will do this with a review of the release script.

Reply:
ok